SeleniumPracticalExercise/PageObjects/LoginPage.cs

```diff
@@ -14,13 +14,13 @@ public LoginPage(IWebDriver driver) : base(driver)
         }
 
         /// <summary>
-        /// Enter username and password 
+        /// Logs in using provided username and password
         /// </summary>
-        /// <param name="username"></param>
-        /// <param name="password"></param>
-        public void Login(string username, string password)
+        /// <param name="userName">we store username.</param>
```

Comment:
"we store username"? What does that even mean? Please look at example in BasePageLocal.cs